index.html

```diff
@@ -208,6 +208,76 @@ <h6>About me</h6>
           </div>
         </div>
       </section>
+
+      <section class="contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/mwkjreek" method="POST">
+          <ul>
+            <li>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                maxlength="30"
+                placeholder="Full name"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                id="mail"
+                name="user_email"
+                placeholder="Email address"
+              />
+            </li>
+            <li>
+              <textarea
+                id="msg"
+                name="user_message"
+                placeholder="write me something..."
+                maxlength="500"
+              ></textarea>
+            </li>
+            <li class="btn-center">
```

Comment:
- [ ] As requested by the previous reviewer, please add `required` property to all the `input` and `textarea` tags to prevent empty form submissions.
E.g: 
```sh
              <input
                type="email"
                id="mail"
                name="user_email"
                placeholder="Email address"
                required
              />
```

- [OPTIONAL] Please remove the `ul` and `li` tags from the `form` tag. Apart from being semantically correct, they are use together to create unordered lists.